cli/pcluster/easyconfig.py

```diff
@@ -82,21 +88,26 @@ def get_regions():
 
 
 @handle_client_exception
-def ec2_conn(aws_region_name):
+def _evaluate_aws_region(aws_region_name):
     if aws_region_name:
         region = aws_region_name
     elif os.environ.get("AWS_DEFAULT_REGION"):
         region = os.environ.get("AWS_DEFAULT_REGION")
     else:
-        region = "us-east-1"
+        region = DEFAULT_REGION
+    return region
+
 
+@handle_client_exception
+def ec2_conn(aws_region_name):
+    region = _evaluate_aws_region(aws_region_name)
     ec2 = boto3.client("ec2", region_name=region)
     return ec2
 
 
 @handle_client_exception
 def list_keys(aws_region_name):
-    conn = ec2_conn(aws_region_name)
+    conn = ec2_conn(_evaluate_aws_region(aws_region_name))
```

Comment:
I removed it from list key so that also list_subnet and list_vpc will still be checked. We don't know if in the future we may change the order of this function and in this way we prevent a future bug.

Reply:
👍 if you think about the function responsibilities it seemed more appropriate for the ec2_conn func to receive an already resolved region value, but I'm fine with that